main-page.css

```diff
@@ -0,0 +1,213 @@
+body {
+    margin: 0;
+}
+.right-pannel {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: space-between;
+    border-radius: 10px 0 0 10px;
+    width: 80px;
+    height: 250px;
+    background-color: white;
```

Comment:
Let's agree to use HEX format for all colors. Should be fixed in all similar places.